marketpulse/main/fxosprice_urls.py

```diff
@@ -4,6 +4,8 @@
 urlpatterns = patterns(
     '{0}.main.views'.format(settings.PROJECT_MODULE),
     url(r'^new/$', 'edit_fxosprice', name='edit_fxosprice'),
-    url(r'^contributions/$', 'all_fxosprice', name='all_fxosprice'),
+    url(r'^all/$', 'all_fxosprice', name='all_fxosprice'),
+    url(r'^all/(?P<user_pk>\d+)/$', 'all_fxosprice', name='user_fxosprice'),
```

Comment:
If this is for all the contributions of a user, I would suggest to use a different schema (eg /username/contributions). A url /all/a_number/ maybe leads to confusion.